npm/src/index.ts

```diff
@@ -127,11 +127,15 @@ export const renderModules = async () => {
       form.addEventListener("submit", async (ev) => {
         ev.preventDefault()
         const body = new URLSearchParams()
+        let selectedItems: string[] = []
         document
           .querySelectorAll("li[aria-checked=true]")
           .forEach((el, _, __) => {
-            body.set(el.id, "on")
+            selectedItems.push(el.id)
           })
+        body.set("Action", (<HTMLInputElement>ev.submitter)?.value)
+        body.set("Ids", selectedItems.join(","))
```

Comment:
I think this means that this won't work without javascript. It doesn't actually work at the moment anyway but that's a separate issue. 
The Ids variable isn't needed if you get the file IDs from the form and I think the `Action` input could be set as a hidden input on the page, or else passed as a request parameter. 

Either way, I think you can discard these changes and find a different way of passing the information through that will work without js.